nengo_loihi/builder/connection.py

```diff
@@ -404,10 +404,10 @@ def build_chip_connection(model, conn):  # noqa: C901
     # Create random number generator
     rng = np.random.RandomState(model.seeds[conn])
 
-    pre_cx = model.objs[conn.pre_obj]['out']
-    post_cx = model.objs[conn.post_obj]['in']
-    assert isinstance(pre_cx, (LoihiBlock, LoihiInput))
-    assert isinstance(post_cx, (LoihiBlock, Probe))
+    pre_compartment = model.objs[conn.pre_obj]['out']
```

Comment:
I think "compartment" here is not accurate, since these are mostly `LoihiBlock` objects, not compartments. I'd be okay with `pre_block` etc., though as you can see in the following line, this can actually be `LoihiBlock` or `LoihiInput`.

Reply:
I basically just replaced "cx" with "compartment" throughout 🤷‍♂